hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -863,6 +863,7 @@ private void instantiateServices(boolean withNewSnapshot) throws IOException {
     omSnapshotManager = new OmSnapshotManager(this);
 
     // Snapshot metrics
+    metrics.setNumSnapshotCacheSize(omSnapshotManager.getSnapshotCacheSize());
```

Comment:
There is no need to set this metric here. It is a cache metic and should be zero on the OM start-up because there is nothing in the cache at that time.

Reply:
Makes sense, removed.